docs/content/concepts/automation/declarative-automation/customizing-automation-conditions.mdx

```diff
@@ -154,8 +154,47 @@ The above conditions can be built into more complex expressions using the follow
   </tbody>
 </table>
 
+### Composite conditions
+
+Finally, there are a set of pre-built conditions which make it easier to construct common combinations of the above conditions.
+
+| Condition                                         | Description                                                                                            |
+| ------------------------------------------------- | ------------------------------------------------------------------------------------------------------ |
+| `AutomationCondition.any_deps_updated`            | Returns true if any dependencies have been updated since the previous tick                             |
+| `AutomationCondition.any_deps_missing`            | Returns true if any dependencies have never been materialized or observed                              |
+| `AutomationCondition.any_deps_in_progress`        | Returns true if any dependencies are part of an in-progress run                                        |
+| `AutomationCondition.all_deps_updated_since_cron` | Returns true if all dependencies have been updated since the latest tick of the provided cron schedule |
+
 ---
 
+## Modifying policies
+
+It's common to have use cases which are similar to pre-built policies, but with minor differences. While it is always possible to simply copy the base implementation and make modifications to it as needed, it can often be simpler to use the `.without()` method to remove the unwanted sub-conditions, or add additional conditions with the `&` operator.
```

Comment:
```suggestion
It's common to have use cases similar to pre-built policies but with minor differences. While it is always possible to copy the base implementation and modify it as needed, it can often be simpler to use the `.without()` method to remove the unwanted sub-conditions or add additional conditions with the `&` operator.
```